src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -112,28 +112,34 @@ public static IMauiHandlersCollection AddMauiControlsHandlers(this IMauiHandlers
 #pragma warning restore CA1416
 
 #if WINDOWS || ANDROID || IOS || MACCATALYST || TIZEN
-			handlersCollection.AddHandler<ListView>(svc => new Handlers.Compatibility.ListViewRenderer(
 #if ANDROID
-				svc.GetRequiredService<Android.Content.Context>()
```

Comment:
I wonder if you are able to get the maui context, and then get the android contextfrom that. The maui context should be registered in the services?

Reply:
But I am not sure _which_ maui context it will be - it may be the root app one, or the latest/nearest...

Reply:
`svc.GetService<IMauContext>()` returns null at this point for some reason?

Reply:
yea, the service passed in here is just the `MauiHandlerFactory` that implements IServiceProvider. It doesn't really have any exciting information or access to the relevant `WrappedServiceProvider`

Currently all these controls are completely broken so just doing this for now to get unbroken. 

This is partially why I was hoping we'd shift to a dependency resolution route. If the `ToHandler` code used `ActivatorUtilities` then we could look at dependency resolution to make this work. 

I think it's still worth seeing if we can modify this factory approach to provide the `WrappedServiceProvider` inside this code block before we print NET8.